lib/livebook_web/live/form_helpers.ex

```diff
@@ -0,0 +1,74 @@
+defmodule LivebookWeb.FormHelpers do
+  @moduledoc """
+  Conveniences for translating and building error messages.
+  """
+
+  use Phoenix.Component
+
+  import Phoenix.HTML.Form
+  import LivebookWeb.LiveHelpers
+
+  @doc """
+  A wrapper for inputs with conveniences.
+  """
+  def input_wrapper(assigns) do
+    assigns = assign_new(assigns, :class, fn -> [] end)
+
+    ~H"""
+    <div
+      phx-feedback-for={input_name(@form, @field)}
+      class={[@class, if(@form.errors[@field], do: "show-errors", else: "")]}
+    >
+      <%= render_slot(@inner_block) %>
+      <%= for error <- Keyword.get_values(@form.errors, @field) do %>
+        <span class="hidden text-red-600 text-sm phx-form-error:block">
+          <%= translate_error(error) %>
+        </span>
+      <% end %>
+    </div>
+    """
+  end
+
+  @doc """
+  Hex color input.
+
+  It expects a phx-click to be assigned to a randomize function.
+  """
+  def hex_color_input(assigns) do
+    ~H"""
+    <div class="flex space-x-4 items-center">
+      <div
+        class="border-[3px] rounded-lg p-1 flex justify-center items-center"
+        style={"border-color: #{input_value(@form, @field)}"}
+      >
+        <div class="rounded h-5 w-5" style={"background-color: #{input_value(@form, @field)}"}></div>
+      </div>
+      <div class="relative grow">
+        <%= text_input(@form, @field,
+          class: "input phx-form-error:border-red-300",
```

Comment:
Would it make sense to add the error case to all `.input`?

Reply:
By using @apply? Good call. I can try.

Reply:
Yeah, we can add to the existing definition in `components.css`.